PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -83,21 +101,27 @@
       "extractKeyPhrasesBatch",
       "analyzeSentimentBatch"
     ],
-    "antiPatternMessage": "Single operation found in loop. This SDK provides a batch operation API, use it to perform multiple actions in a single request: "
+    "antiPatternMessage": "Single operation found in loop. This SDK provides a batch operation API, use it to perform multiple actions in a single request: ",
+    "recommendationText": "A a batch operation API exists in the SDK that can handle multiple actions in one request.",
+    "recommendationLink": ""
```

Comment:
will ask for link from Shawn

Reply:
Is this still pending?

Reply:
yes-- I messaged him about it, so just waiting to hear back